/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/pettingzoo/utils/conversions.py:320: UserWarning: The `infos` dictionary returned by `env.reset` was empty. OverwritingAgent IDs will be used as keys
  warnings.warn(
100
update number 1 total reward 208.0
SPS: 24
SPS: 30
SPS: 34
update number 4 total reward 125.0
SPS: 33
SPS: 34
SPS: 36
update number 7 total reward 198.0
SPS: 35
SPS: 35
SPS: 36
update number 10 total reward 155.0
SPS: 35
SPS: 36
SPS: 37
update number 13 total reward 214.0
SPS: 36
SPS: 36
SPS: 37
update number 16 total reward 187.0
SPS: 36
SPS: 37
SPS: 37
update number 19 total reward 290.0
SPS: 36
SPS: 37
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/Users/Henry_1/Documents/Part B/Career ML Stuff/meltingpot/examples/pettingzoo/cleanrl_train.py", line 332, in <module>
    next_obs, reward, done, info = envs.step(action.cpu().numpy())
                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/stable_baselines3/common/vec_env/base_vec_env.py", line 206, in step
    return self.step_wait()
           ^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/supersuit/vector/sb3_vector_wrapper.py", line 26, in step_wait
    observations, rewards, terminations, truncations, infos = self.venv.step_wait()
                                                              ^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/supersuit/vector/concat_vec_env.py", line 76, in step_wait
    return self.step(self._saved_actions)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/supersuit/vector/concat_vec_env.py", line 84, in step
    venv.step(
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/supersuit/vector/markov_vector_wrapper.py", line 95, in step
    observations, reset_infs = self.reset()
                               ^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/supersuit/vector/markov_vector_wrapper.py", line 57, in reset
    _observations, infos = self.par_env.reset(seed=seed, options=options)
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/pettingzoo/utils/conversions.py", line 179, in reset
    self.aec_env.reset(seed=seed, options=options)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/supersuit/utils/base_aec_wrapper.py", line 31, in reset
    super().reset(seed=seed, options=options)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/pettingzoo/utils/wrappers/order_enforcing.py", line 118, in reset
    super().reset(seed=seed, options=options)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/pettingzoo/utils/wrappers/base.py", line 38, in reset
    self.env.reset(seed=seed, options=options)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/pettingzoo/utils/wrappers/order_enforcing.py", line 118, in reset
    super().reset(seed=seed, options=options)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/pettingzoo/utils/wrappers/base.py", line 38, in reset
    self.env.reset(seed=seed, options=options)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/pettingzoo/utils/conversions.py", line 306, in reset
    self._observations, self.infos = self.env.reset(seed=seed, options=options)
                                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/supersuit/generic_wrappers/utils/shared_wrapper_util.py", line 115, in reset
    observations, infos = super().reset(seed=seed, options=options)
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/pettingzoo/utils/wrappers/base_parallel.py", line 23, in reset
    return self.env.reset(seed=seed, options=options)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/pettingzoo/utils/conversions.py", line 179, in reset
    self.aec_env.reset(seed=seed, options=options)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/supersuit/utils/base_aec_wrapper.py", line 31, in reset
    super().reset(seed=seed, options=options)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/pettingzoo/utils/wrappers/order_enforcing.py", line 118, in reset
    super().reset(seed=seed, options=options)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/pettingzoo/utils/wrappers/base.py", line 38, in reset
    self.env.reset(seed=seed, options=options)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/pettingzoo/utils/wrappers/order_enforcing.py", line 118, in reset
    super().reset(seed=seed, options=options)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/pettingzoo/utils/wrappers/base.py", line 38, in reset
    self.env.reset(seed=seed, options=options)
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/pettingzoo/utils/conversions.py", line 306, in reset
    self._observations, self.infos = self.env.reset(seed=seed, options=options)
                                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/Henry_1/Documents/Part B/Career ML Stuff/meltingpot/examples/pettingzoo/utils.py", line 76, in reset
    self.agents = self.possible_agents[:]
               ^^^^^^^^^^^^^^^^^
  File "/Users/Henry_1/Documents/Part B/Career ML Stuff/meltingpot/meltingpot/utils/substrates/substrate.py", line 68, in reset
  File "/Users/Henry_1/Documents/Part B/Career ML Stuff/meltingpot/meltingpot/utils/substrates/wrappers/base.py", line 40, in reset
  File "/Users/Henry_1/Documents/Part B/Career ML Stuff/meltingpot/meltingpot/utils/substrates/wrappers/collective_reward_wrapper.py", line 54, in reset
  File "/Users/Henry_1/Documents/Part B/Career ML Stuff/meltingpot/meltingpot/utils/substrates/wrappers/base.py", line 40, in reset
  File "/Users/Henry_1/Documents/Part B/Career ML Stuff/meltingpot/meltingpot/utils/substrates/wrappers/base.py", line 40, in reset
  File "/Users/Henry_1/Documents/Part B/Career ML Stuff/meltingpot/meltingpot/utils/substrates/wrappers/multiplayer_wrapper.py", line 134, in reset
  File "/Users/Henry_1/Documents/Part B/Career ML Stuff/meltingpot/meltingpot/utils/substrates/wrappers/base.py", line 40, in reset
  File "/Users/Henry_1/Documents/Part B/Career ML Stuff/meltingpot/meltingpot/utils/substrates/wrappers/observables_wrapper.py", line 48, in reset
  File "/Users/Henry_1/Documents/Part B/Career ML Stuff/meltingpot/meltingpot/utils/substrates/wrappers/base.py", line 40, in reset
  File "/Users/Henry_1/Documents/Part B/Career ML Stuff/meltingpot/meltingpot/utils/substrates/wrappers/reset_wrapper.py", line 41, in reset
  File "/Users/Henry_1/Documents/Part B/Career ML Stuff/meltingpot/meltingpot/utils/substrates/builder.py", line 182, in build_environment
ValueError: Failed to open file '/Users/Henry_1/Documents/Part B/Career ML Stuff/meltingpot/meltingpot/lua/levels/commons_harvest/init.lua'